hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,592 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "spi_host",
+  clock_primary: "clk_i",
+  other_clock_list: [ "clk_core_i" ],
+  reset_primary: "rst_ni",
+  other_reset_list: [ "rst_core_ni" ],
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
+  param_list: [
+    { name: "ByteOrder",
```

Comment:
@tjaychen: General Comment: ([Carried over from 5077](https://github.com/lowRISC/opentitan/pull/5077#discussion_r571301044)):  Is this better as a parameter or a configuration option?